### PR TITLE
Base location finder

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@gsvr review please. Probably better to pull local.

File: django_refugeerights/runtests.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+export DJANGO_SETTINGS_MODULE="django_refugeerights.testsettings"
+cd django_refugeerights
```
Comment (gsvr) [inline django_refugeerights/runtests.sh]: I think this `cd` should be removed - manage.py is in the same directory.

Comment (imsickofmaps) [inline django_refugeerights/runtests.sh:4]: No, the file is called from the root by travis so it's right

File: django_refugeerights/requirements-dev.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+pytest
+pytest-cov
+pytest-django
+flake8
```
Comment (gsvr) [inline django_refugeerights/requirements-dev.txt]: EOF newline

File: django_refugeerights/locationfinder/tasks.py [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+        code.
+        """
+
+    def vumi_client(self):
+        return HttpApiSender(
+            account_key=settings.VUMI_GO_ACCOUNT_KEY,
+            conversation_key=settings.VUMI_GO_CONVERSATION_KEY,
+            conversation_token=settings.VUMI_GO_ACCOUNT_TOKEN
+        )
+        # return LoggingSender('go_http.test')
```
Comment (gsvr) [inline django_refugeerights/locationfinder/tasks.py]: Rogue comment?

**Comment by gsvr** [comment]
:+1:

**Pull request merged**